tasks/redhat.yml

```diff
@@ -66,20 +66,5 @@
        insertafter: '^\[Service\]'
        firstmatch: yes
        state: present
-    notify: restart_collectd
-
-  - name: enable and start collectd
-    service: name=collectd state=started enabled=yes
-    when: ansible_connection != 'chroot'
```

Comment:
These were added in 348846b88a2b85db4ff8f6d39b59f0e73c036f6b "Make role work with chroot images".

Any idea if the role for this use case would break again from this change?

Reply:
The handler `restart_collectd` should take care of this IMHO.

Reply:
Since we're not running this in a chroot ourselves, I wonder if @jabl or @mhakala could chime in for the review of this one?

Reply:
I re-added the not restarting when using chroot.